lib/db-utils/client.js

```diff
@@ -92,7 +92,7 @@ export function getSuitesTableRows(db) {
 }
 
 export function getTableRows(db, tableName) {
-    const [databaseRows] = db.exec(selectAllQuery(tableName));
+    const databaseRows = db.exec(selectAllQuery(tableName))[0];
```

Comment:
everywhere "[0]" at the end of "exec" means "first executed statement" because "sql.js" supports executing multiple js statements in "exec"